lib/controller.js

```diff
@@ -800,7 +800,7 @@ controller.printAvailableUpdates = function() {
   });
 };
 
-controller.update = function(opts) {
+controller.update = function(opts) {  
```

Comment:
This shouldn't be here ;)

You need to run `grunt` locally, which will clean up things like that for you and also ensures that all the linting and tests are run. Ping me after you've had a chance to do that.